metadata-ingestion/src/datahub/ingestion/source/fivetran/fivetran_log_api.py

```diff
@@ -120,15 +152,22 @@ def _get_jobs_list(self, connector_id: str) -> List[Job]:
     def _get_user_name(self, user_id: Optional[str]) -> Optional[str]:
         if not user_id:
             return None
-        user_details = self._query(FivetranLogQuery.get_user_query(user_id=user_id))[0]
+        user_details = self._query(
+            self.fivetran_log_query.get_user_query(user_id=user_id)
+        )[0]
         return (
             f"{user_details[Constant.GIVEN_NAME]} {user_details[Constant.FAMILY_NAME]}"
         )
 
-    def get_connectors_list(self) -> List[Connector]:
+    def get_connectors_list(
```

Comment:
I would rather keep the connector_pattern logic in the calling method instead of here

If we want to keep it here, let's make it more clear

```suggestion
    def get_allowed_connectors_list(
```